src/ui/overlay.html

```diff
@@ -18,8 +18,8 @@
 
     .item-slot {
       position: relative;
-      height: 100%;
-      padding-bottom: 20vh;
+      height: 50%;
+      top: 20%;
```

Comment:
Why do we have to resize the overlay content ? 🤔

Reply:
Had to resize because of how the height of the images worked with the fixed height of the window. Since you've proposed to just show the error message on top of the icons, this is indeed useless.